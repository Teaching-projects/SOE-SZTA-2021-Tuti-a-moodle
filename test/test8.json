{
    "name": "John",
    "lore": "Description for John",
    "health_points": 100,
    "attack_cooldown": 2,
    "damage": 50,
    "defense": 10
}
