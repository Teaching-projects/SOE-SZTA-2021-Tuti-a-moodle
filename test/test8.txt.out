====================
John - HP: 100.0, DMG: 50.0, DEF: 10.0
Bob - HP: 90.0, DMG: 60.0, DEF: 0.0
====================
The Battle begins.
John and Bob clash!
====================
The Battle is over.
Bob won!
